Lib/test/test_os.py

```diff
@@ -1026,6 +1026,74 @@ def test_iter_error_when_changing_os_environ_items(self):
     def test_iter_error_when_changing_os_environ_values(self):
         self._test_environ_iteration(os.environ.values())
 
+    def test_or_operator(self):
+        overridden_key = '_TEST_VAR_'
+        os.environ[overridden_key] = 'original_value'
+
+        new_vars_dict = {'_A_': '1', '_B_': '2', overridden_key: '3'}
+        expected = os.environ.copy()
```

Comment:
Maybe `dict(os.environ)` to emphasize that we expect the actual value to be a plain dict?

Reply:
Done.